articles/azure-functions/functions-monitoring.md

```diff
@@ -391,13 +391,13 @@ namespace functionapp0915
          
             // Track an Event
             var evt = new EventTelemetry("Function called");
-            UpdateTelemetryContext(evt.Context, context, userName);
-            telemetry.TrackEvent(evt);
+            UpdateTelemetryContext(evt.Context, context, name);
+            telemetryClient.TrackEvent(evt);
             
             // Track a Metric
             var metric = new MetricTelemetry("Test Metric", DateTime.Now.Millisecond);
-            UpdateTelemetryContext(metric.Context, context, userName);
-            telemetry.TrackMetric(metric);
+            UpdateTelemetryContext(metric.Context, context, name);
+            telemetryClient.TrackMetric(metric);
             
             // Track a Dependency
             var dependency = new DependencyTelemetry
```

Comment:
BTW, why is Name here a URL ("GET api/planets/1/")?  Could/should it be descriptive text like the other telemetry types ("Test Metric", "Function called")

Reply:
That came from @lmolkova -- it's how App Insights names dependencies so we wanted it to be consistent. See the comment here: https://github.com/MicrosoftDocs/azure-docs/pull/4998#pullrequestreview-99094581